### PR TITLE
OM-863 | Store OIDC callback/redirect URI for GDPR API authentication

### PR DESCRIPTION
**Pull request by charn**
This PR adds environment and configuration variable for storing the callback URI related to the GDPR API's OIDC authorization code retrieval. Backend needs this URI when exchanging the authorization code into an access token.

### CODE REVIEW

**Comment by akikoskinen** [comment]
This just sets a setting. Where is this setting used? Or is that part coming?

**Comment by charn** [comment]
That part is coming on the next PR.

File: open_city_profile/settings.py [hunk 1]
```diff
@@ -57,6 +57,7 @@
     USE_X_FORWARDED_HOST=(bool, None),
     CSRF_TRUSTED_ORIGINS=(list, []),
     TEMPORARY_PROFILE_READ_ACCESS_TOKEN_VALIDITY_MINUTES=(int, 2 * 24 * 60),
+    GDPR_AUTH_CALLBACK_URL=("", None),
```
Comment (akikoskinen) [inline open_city_profile/settings.py:60]: Shouldn't this be `(str, None)` or perhaps `(str, "")`.

**Review by akikoskinen — CHANGES_REQUESTED** [review]

File: open_city_profile/settings.py (hunk 1 above)
Comment (charn) [reply open_city_profile/settings.py:60]: It should. :sweat_smile:

Comment (charn) [reply open_city_profile/settings.py:60]: Okay, fixed!

**Review by akikoskinen — APPROVED** [review]

**Pull request merged**